client/jest.config.js

```diff
@@ -0,0 +1,29 @@
+const { pathsToModuleNameMapper } = require('ts-jest');
```

Comment:
Why was this change needed? (naive question, I'm still learning about setting up environments)

Reply:
We need the mapper otherwise Jest gets confused about the tsconfig paths (it can't resolve things like "@shared/data-access/user/user.service" on its own.